### ISSUE TITLE
RCS0060 doesn't report an error if usings are placed after the namespace declaration

### ISSUE THREAD

**Issue by xt0rted** [JosefPihrt/Roslynator#1011]
**Product and Version Used**: 4.2.0

**Steps to Reproduce**:

```.editorconfig
roslynator_blank_line_after_file_scoped_namespace_declaration = true

# RCS0060: Add/remove line after file scoped namespace declaration
dotnet_diagnostic.RCS0060.severity = error
```

```csharp
namespace MyApp;
using MyApp.Things;

public class Stuff
{
}
```

**Actual Behavior**:

No error is reported.

**Expected Behavior**:

For an error to be reported and the suggested fix to be:

```csharp
namespace MyApp;

using MyApp.Things;

public class Stuff
{
}
```
